DaisyMicheal/Ecommerce_web
# Implement cart page layout and functionality

**Pull Request Description for Cart Page**

**What does this PR do?**
Create and implement the cart page for the e-commerce platform.

**Description of Task to be completed:**
- Design and implement the cart page layout, displaying selected products, quantities, and pricing.
- Implement functionality to adjust item quantities directly from the cart page.
- Include a feature to remove items from the cart.
- Display the total price and update it dynamically as items are added, removed, or quantities are changed.
- Add a "Proceed to Checkout" button that redirects users to the payment or checkout page.
- Ensure that the cart information is persistent across sessions, using local storage or backend integration.
- Provide visual feedback for users when items are added to or removed from the cart.
- Ensure the cart page is fully responsive and works well across different devices.
- Write unit tests to cover the cart's functionality, including adding/removing items and adjusting quantities.

**How should this be manually tested?**
- Pull the branch and install dependencies if necessary.
- Navigate to the cart page and add items to the cart from the product pages.
- Adjust item quantities and verify that the total price updates correctly.
- Remove items from the cart and ensure they are removed from the list and the total price is updated.
- Test the "Proceed to Checkout" button to ensure it redirects to the correct page.
- Refresh the page or close and reopen the browser to verify that the cart data persists.
- Run the unit tests to ensure they pass.

**Any background context you want to provide?**
None

**What are the relevant Pivotal Tracker stories?**
#188043127

**Screenshots (if appropriate):**
[image]

**Questions:**
None
